test/NuGet.Tests.Apex/NuGet.Tests.Apex.Daily/NuGetEndToEndTests/NuGetUITestCase.cs

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Test.Apex.VisualStudio.Solution;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using NuGet.Test.Utility;
+
+namespace NuGet.Tests.Apex.Daily
+{
+    [TestClass]
+    public class NuGetUITestCase : SharedVisualStudioHostTestClass
+    {
+        private const string TestPackageName = "Contoso.A";
+        private const string TestPackageVersionV1 = "1.0.0";
+
+        private readonly SimpleTestPathContext _pathContext = new SimpleTestPathContext();
+
+        public NuGetUITestCase()
+            : base()
+        {
+        }
+
+        [TestMethod]
+        [Timeout(DefaultTimeout)]
+        public async Task TestPipeline()
```

Comment:
I'm guessing this is something that gets deleted eventually correct?

I'd just keep the test as simple as possible, since it seems to function as a dummy test.

Reply:
> I'm guessing this is something that gets deleted eventually correct?
> 
> I'd just keep the test as simple as possible, since it seems to function as a dummy test.

Yes, it was added just to verify the new _NuGet.Tests.Apex.Daily.runsettings_ works well in Apex daily test pipeline, it will be removed when a valid test case is added.